test/jdk/java/awt/Robot/ScreenCaptureRobotTest.java

```diff
@@ -113,28 +117,50 @@ private static boolean compareImages(BufferedImage capturedImg,
         int realPixel;
         int imgWidth;
         int imgHeight;
-        boolean result = true;
 
         imgWidth = capturedImg.getWidth(null);
         imgHeight = capturedImg.getHeight(null);
 
+        if (imgWidth != realImg.getWidth()
+                || imgHeight != realImg.getHeight()) {
+            System.out.println(
+                    "Captured image and real image widths are different");
+            return false;
+        }
+
         for (int i = 0; i < (imgWidth); i++) {
             for (int j = 0; j < (imgHeight); j++) {
```

Comment:
No parentheses around `imgWidth` and `imgHeight`

for (int i = 0; i < imgWidth; i++) {
            for (int j = 0; j < imgHeight; j++) {